deployments/ticdc/docker-compose/docker-compose-kafka-integration.yml

```diff
@@ -11,8 +11,9 @@ services:
     ports:
       - "9092:9092"
     environment:
-      KAFKA_MESSAGE_MAX_BYTES: 1073741824
-      KAFKA_REPLICA_FETCH_MAX_BYTES: 1073741824
+      KAFKA_MESSAGE_MAX_BYTES: 11534336
+      KAFKA_REPLICA_FETCH_MAX_BYTES: 11534336
```

Comment:
Nit: Could you add some comments about these numbers?

Reply:
added!